Octokit.Tests.Integration/fixtures/RepositoriesHooksCollection.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Xunit;
+
+namespace Octokit.Tests.Integration.fixtures
+{
+    [CollectionDefinition("Repositories Hooks Collection")]
+    public class RepositoriesHooksCollection : ICollectionFixture<RepositoriesHooksFixture>
```

Comment:
Yeah. There were two reasons for why I want to use collection fixtures. Never tried them before, but I do like them and would like to have them in other test frameworks. :smile:

The first and foremost reason is that for all/alot of my tests I only want to create my repositories and other readonly states once for all tests, so to minimize the times I am hitting the github Api. If I run all the tests in Octokit I will be throttled by github.

The second reason is that it helps encapsulate the creation and disposal logic, though it is a bit indirect and requires people to know them.

I looked at the code and not having read it for 2-3 weeks, I have some suggestions which I would like to run by you guys.

I define the name of the collection as a constant string in the collection class:
```C#
    [CollectionDefinition(CollectionDefinitionName)]
    public class RepositoriesHooksCollection : ICollectionFixture<RepositoriesHooksFixture>
    {
      public const string CollectionDefinitionName = "Repositories Hooks Collection";
    }
```

Which I then use in the 'CollectionDefinition' attribute. On each test class instead of writing the string name of the collection, I use the constant string defined in the collection class:
```C#
        [Collection(RepositoriesHooksCollection.CollectionDefinitionName)]
        public class TheGetAllMethod
        {
            readonly RepositoriesHooksFixture _fixture;

            public TheGetAllMethod(RepositoriesHooksFixture fixture)
            {
```
That should make it a bit less brittle than before.

Furthermore the `RepositoriesHooksCollection` class is very small and very closely tied to the `RepositoriesHooksFixture` class. Would it be an idea to move the collection class into the same file as the fixture class. I am not normally a big fan of having two class inside the same file.